zookeeper-server/src/main/java/org/apache/zookeeper/ZooKeeper.java

```diff
@@ -2665,6 +2668,68 @@ public void getChildren(String path, boolean watch, Children2Callback cb,
         getChildren(path, watch ? watchManager.defaultWatcher : null, cb, ctx);
     }
 
+    /**
+     * Synchronously gets all the ephemeral nodes  created by this session.
+     *
+     * @since 3.5.0
```

Comment:
This should go to 3.6.0.
3.5 branch is almost close

Reply:
Updated with 3.6.0